File: MODUL03/PraktikumC/PRAK305-231081711004-AbdurrahmanGilangHarjuna.c
#include <stdio.h>

int main() {
    int detik, jam, menit, sisa_detik;

    printf("Masukkan detik: ");
    scanf("%d", &detik);

    jam = detik / 3600;
    sisa_detik = detik % 3600;
    menit = sisa_detik / 60;
    sisa_detik = sisa_detik % 60;

    if (jam > 24) {
        int hari = jam / 24; 
        jam = jam % 24;      
        printf("%d hari %02d:%02d:%02d\n", hari, jam, menit, sisa_detik);
    } else {
        printf("%02d:%02d:%02d\n", jam, menit, sisa_detik);
    }

    return 0;
}

File: MODUL03/PraktikumPy/PRAK305-231081711004-AbdurrahmanGilangHarjuna.py
detik = int(input("Masukkan detik: "))

jam = detik // 3600
sisadetik = detik % 3600
menit = sisadetik // 60
sisadetik = sisadetik % 60

if jam > 24:
    hari = jam // 24
    jam = jam % 24
    print(f"{hari} hari {jam:02d}:{menit:02d}:{sisadetik:02d}")
else:
    print(f"{jam:02d}:{menit:02d}:{sisadetik:02d}")
